src/framework/ui/uitypes.h

```diff
@@ -162,14 +162,16 @@ struct UiAction
     std::string shortcut;
 
     UiAction() = default;
-    UiAction(const actions::ActionCode& code, UiContext ctx, const QString& title, Checkable ch = Checkable::No)
+    UiAction(const actions::ActionCode& code, UiContext ctx, Checkable ch = Checkable::No)
+        : code(code), context(ctx), checkable(ch) {}
+    UiAction(const actions::ActionCode& code, UiContext ctx, const char* title, Checkable ch = Checkable::No)
         : code(code), context(ctx), title(title), checkable(ch) {}
-    UiAction(const actions::ActionCode& code, UiContext ctx, const QString& title, const QString& desc, Checkable ch = Checkable::No)
+    UiAction(const actions::ActionCode& code, UiContext ctx, const char* title, const char* desc, Checkable ch = Checkable::No)
```

Comment:
Better to use std::string_view instead of const char*

Reply:
no, not in this case :)
typically we do
```
const UiActionList ApplicationUiActions::m_actions = {
    UiAction("quit",
             mu::context::UiCtxAny,
             QT_TRANSLATE_NOOP("action", "Quit")
             ),
    UiAction("fullscreen",
             mu::context::UiCtxAny,
             QT_TRANSLATE_NOOP("action", "Full Screen"),
             QT_TRANSLATE_NOOP("action", "Full screen"),
             Checkable::Yes
             ),
```
here `QT_TRANSLATE_NOOP("action", "Esc")` just `const char*`